Update phone number assignment cmdlets

JensTrier opened the pull request:
We are changing the way phone numbers are assigned to Microsoft Teams users and resources. This PR adds deprecation notice to the existing cmdlets and points readers to using the new cmdlets.

Also adds a new example to set-csphonenumberassignment.

opbld30 commented:
Docs Build status updates of commit _[ca993e0](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/ca993e04c06cd1e57a5f918945f87cbcaa775d26)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/new-csonlinebulkassignmentinput?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlineapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8834) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C1%5C20%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202201200912572669-8834%5Cworkflow_report.html?sv=2020-08-04&se=2022-02-20T09%3A15%3A46Z&sr=b&sp=r&sig=addaLmxOBxvHGREd4xG%2FN7MWbUXJPDaXENMzHLtPBac%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

JohanFreelancer9 commented on skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md:
```diff
@@ -15,6 +15,9 @@ ms.reviewer:
 ## SYNOPSIS
 Provide the topic introduction here.
 
+> [!NOTE]
+> This cmdlet will be deprecated.
```
```suggestion
**Note**: This cmdlet will be deprecated.
```

JohanFreelancer9 commented on skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md:
```diff
@@ -15,6 +15,9 @@ ms.reviewer:
 ## SYNOPSIS
 Updates an application instance in Azure Active Directory. 
 
+> [!NOTE]
+> This cmdlet will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```
```suggestion
**Note**:  This cmdlet will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```

JohanFreelancer9 commented on skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md:
```diff
@@ -6,15 +6,18 @@ title: Set-CsOnlineVoiceApplicationInstance
 ## SYNOPSIS
 The `Set-CsOnlineVoiceApplicationInstance` modifies an application instance in Azure Active Directory.
 
+> [!NOTE]
+> This cmdlet will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```
```suggestion
**Note**: This cmdlet will be deprecated. Start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) instead.

```

JohanFreelancer9 commented on skype/skype-ps/skype/Set-CsOnlineVoiceUser.md:
```diff
@@ -15,6 +15,9 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Set-CsOnlineVoiceUser` cmdlet to set the PSTN specific parameters (like telephone numbers and emergency response locations.)
 
+> [!NOTE]
+> This cmdlet will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```
```suggestion
**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
```

JohanFreelancer9 commented on skype/skype-ps/skype/Set-CsUser.md:
```diff
@@ -17,6 +17,9 @@ Modifies Skype for Business properties for an existing user account.
 Properties can be modified only for accounts that have been enabled for use with Skype for Business.
 This cmdlet was introduced in Lync Server 2010.
 
+> [!NOTE]
+> Using this cmdlet for Microsoft Teams users will be deprecated. For details, please see individual parameters below.
+> Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```
```suggestion
**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
```

JohanFreelancer9 commented on skype/skype-ps/skype/Set-CsUser.md:
```diff
@@ -158,6 +161,8 @@ Accept wildcard characters: False
 Indicates whether the user has been enabled for Enterprise Voice, which is the Microsoft implementation of Voice over Internet Protocol (VoIP).
 With Enterprise Voice, users can make telephone calls using the Internet rather than using the standard telephone network.
 
+> [!NOTE]
+> Using this parameter for Microsoft Teams users will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment)
```
```suggestion
**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
```

JohanFreelancer9 commented on skype/skype-ps/skype/Set-CsUser.md:
```diff
@@ -176,6 +181,9 @@ Accept wildcard characters: False
 When set to True, enables a user's voice mail calls to be routed to a hosted version of Microsoft Exchange Server.
 In addition, setting this option to True enables Skype for Business users to directly place a call to another user's voice mail.
 
+> [!NOTE]
+> Using this parameter for Microsoft Teams users will be deprecated and it is not required anymore to set it for Microsoft Teams users.
+
```
```suggestion
**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
```

JohanFreelancer9 commented on skype/skype-ps/skype/Set-CsUser.md:
```diff
@@ -407,6 +415,9 @@ Depending on your setup, those two numbers could actually be different.
 For example, in some organizations dialing 1-425-555-1297 routes your call to an Exchange Auto Attendant.
 Conversely, dialing just the extension (51297) or using Skype for Business to dial the number 1-425-555-1297 extension 51297 will route your call directly to the user.
 
+> [!NOTE]
+> Using this parameter for Microsoft Teams users will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```
```suggestion
**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
```

JensTrier replied:
@JohanFreelancer9 the cmdlet Set-CsUser itself will stay, but not for Teams. So it's really the parameters that only will be deprecated

opbld30 commented:
Docs Build status updates of commit _[cb23218](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/cb2321840260c26908fde0c09166569270eefc94)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/new-csonlinebulkassignmentinput?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlineapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8834) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C1%5C21%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202201210801338068-8834%5Cworkflow_report.html?sv=2020-08-04&se=2022-02-21T08%3A04%3A27Z&sr=b&sp=r&sig=vcHtHBFQitNxSaNUx2cqajqoBLx0mj8PuRQLHNFU63g%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld31 commented:
Docs Build status updates of commit _[30be079](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/30be0796cc8d4c1339abdaf4c682d84b6bc47d00)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/new-csonlinebulkassignmentinput?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlineapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8834) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C1%5C21%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202201210801470493-8834%5Cworkflow_report.html?sv=2020-08-04&se=2022-02-21T08%3A07%3A16Z&sr=b&sp=r&sig=vx%2BhNb1xxl%2FSBOT9rI3YO9w5%2Fukd1L28srNEXo%2FBrCs%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld31 commented:
Docs Build status updates of commit _[7f27920](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/7f27920cec57dd01a35a4e51a2ff1b76b8128614)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/new-csonlinebulkassignmentinput?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlineapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8834) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C1%5C21%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202201210805278917-8834%5Cworkflow_report.html?sv=2020-08-04&se=2022-02-21T08%3A09%3A56Z&sr=b&sp=r&sig=6YsPI1aqEKmZ%2BamaF9050ODbZiUy4oqWdNY6Yu2tSOk%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

JohanFreelancer9 reviewed (changes requested):
Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

JensTrier commented:
> Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

@JohanFreelancer9 please see comment above on set-csuser. That cmdlet stays, since it is used for SfB On-prem. It's only the parameters used for Teams that will be deprecated. That's why I had the note on the parameters

JohanFreelancer9 reviewed (changes requested):
@JensTrier, thank you for the clarification. I have changed my suggestions accordingly. Could you please commit the remaining three changes if you agree with them? Many thanks.

opbld32 commented:
Docs Build status updates of commit _[2f87429](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/2f874295e91712ca784fae028ac1cce50a983878)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/new-csonlinebulkassignmentinput?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlineapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8834) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C1%5C21%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202201210826077195-8834%5Cworkflow_report.html?sv=2020-08-04&se=2022-02-21T08%3A28%3A58Z&sr=b&sp=r&sig=Ge3B6LkVf9f5nbTx%2Br%2FqBWXzC6OirPxFjnC%2FQ4Q9cZQ%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

JensTrier commented:
> @JensTrier, thank you for the clarification. I have changed my suggestions accordingly. Could you please commit the remaining three changes if you agree with them? Many thanks.

Thanks, I did your changes. I also changed the note on top of Set-CsUser. Please have a look.

opbld30 commented:
Docs Build status updates of commit _[5f69370](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/5f69370ac7b307f9bc2f1300e87ed7cfd8502e51)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/new-csonlinebulkassignmentinput?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlineapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8834) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C1%5C21%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202201210826233598-8834%5Cworkflow_report.html?sv=2020-08-04&se=2022-02-21T08%3A31%3A52Z&sr=b&sp=r&sig=N9ONGwQwcdAXZcr8oFLeuAcqjIUcd5wFsOgobaPy%2BxI%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

JohanFreelancer9 reviewed (approved):
Thank you @JensTrier.

Scifire commented:
@JensTrier Not sure if this is the right place, please redirect me to the right one in case it´s wrong.

I´m missing the parameter "-HostedVoiceMail $true" which I´ve to set in "Set-CsUser". There is no note how this can be set or if it´s not needed anymore to set this.

JensTrier commented:
> @JensTrier Not sure if this is the right place, please redirect me to the right one in case it´s wrong.
> 
> I´m missing the parameter "-HostedVoiceMail $true" which I´ve to set in "Set-CsUser". There is no note how this can be set or if it´s not needed anymore to set this.

Thanks for the feedback! I added this note to Set-CsUser for the HostedVoiceMail parameter: Using this parameter will be deprecated for Teams users and it is not required anymore to set it for Microsoft Teams users.

Don't you see that?

Scifire commented:
> > @JensTrier Not sure if this is the right place, please redirect me to the right one in case it´s wrong.
> > I´m missing the parameter "-HostedVoiceMail $true" which I´ve to set in "Set-CsUser". There is no note how this can be set or if it´s not needed anymore to set this.
> 
> Thanks for the feedback! I added this note to Set-CsUser for the HostedVoiceMail parameter: Using this parameter will be deprecated for Teams users and it is not required anymore to set it for Microsoft Teams users.
> 
> Don't you see that?

Ahh sorry, my mistake. Didn´t checked the description of the parameter in the PR. Okay then everything should be explained and fine. Thanks :)

opbld33 commented:
Docs Build status updates of commit _[4d3bb2c](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/4d3bb2c1efc9c133183bfae85883e96641faf738)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/new-csonlinebulkassignmentinput?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlineapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8834) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C1%5C21%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202201211150101185-8834%5Cworkflow_report.html?sv=2020-08-04&se=2022-02-21T11%3A52%3A55Z&sr=b&sp=r&sig=luvuAmCUy1sZWup2%2F7TS242q23qS9fjZUuzwx2qqYbQ%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

yogkumgit commented:
@dariomws @tseward @get-itips

yogkumgit commented:
@tiburd Please merge this PR. Thanks

opbld30 commented:
Docs Build status updates of commit _[8882ef3](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/8882ef353f647a8652b7dfc13837b5bff853bf84)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/New-CsOnlineBulkAssignmentInput.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/new-csonlinebulkassignmentinput?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlineapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceApplicationInstance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceapplicationinstance?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsOnlineVoiceUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsOnlineVoiceUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[skype/skype-ps/skype/Set-CsUser.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/skype/skype-ps/skype/Set-CsUser.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csuser?view=skype-ps&branch=pr-en-us-8834) (skype-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/DeprecatedPhoneNumberCmdlets/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8834) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C1%5C25%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202201252239493493-8834%5Cworkflow_report.html?sv=2020-08-04&se=2022-02-25T22%3A42%3A37Z&sr=b&sp=r&sig=iTVX9ks%2BgNJvO8auBDSbnxvwH36dCWKR8huIpMg6cvI%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
